Sources_v3/Query/UserListQuery.swift

```diff
@@ -25,24 +25,29 @@ public struct UserListQuery: Encodable {
     
     /// Init a users query.
     /// - Parameters:
-    ///   - filter: a users filter.
+    ///   - filter: a users filter. Empty filter will return all users.
     ///   - sort: a sorting list for users.
     ///   - pagination: a users pagination.
     public init(
-        filter: Filter,
+        filter: Filter? = nil,
         sort: [Sorting<UserListSortingKey>] = [],
         pagination: Pagination = [.usersPageSize]
     ) {
-        if case .none = filter {}
         self.filter = filter
         self.sort = sort
         self.pagination = pagination
     }
     
     public func encode(to encoder: Encoder) throws {
         var container = encoder.container(keyedBy: CodingKeys.self)
-        try container.encode(filter, forKey: .filter)
-        
+        switch filter {
+        case let .some(value): try container.encode(value, forKey: .filter)
+        case .none:
+            // Backend expects empty object for "filter_conditions" in case no filter specified.
+            struct EmptyObject: Encodable {}
+            try container.encode(EmptyObject(), forKey: .filter)
```

Comment:
I don't like this part 🙂
Do you know more elegant way to simulate empty object?
Backend refuses to accept `null` or empty string.

Reply:
Yeah, this is unfortunately needed. You can do something like:
```swift
try container.encode([String](), forKey: .filter)
```
But to honest, I find you solution easier to understand and I don't mind having it in the codebase.